sysutils/diffutils/Portfile

```diff
@@ -1,32 +1,31 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
+PortSystem                  1.0
 
-PortGroup           compiler_blacklist_versions 1.0
+PortGroup                   compiler_blacklist_versions 1.0
 
-name                diffutils
-version             3.10
-revision            0
+name                        diffutils
+version                     3.10
+revision                    1
```

Comment:
The revision shouldn't be increased when fixing a problem that prevented installation.